pep-0249.txt

```diff
@@ -5,7 +5,7 @@ Discussions-To: db-sig@python.org
 Status: Final
```

Comment:
If this PEP is `Informational`, shouldn't it be `Active`, givens the guidance within is presumably still valid, up to date and in-use rather than historical and frozen?